lib/whois/parsers/whois.nic.club.rb

```diff
@@ -0,0 +1,117 @@
+#--
+# Ruby Whois
+#
+# An intelligent pure Ruby WHOIS client and parser.
+#
+# Copyright (c) 2009-2015 Simone Carletti <weppos@weppos.net>
+#++
+
+require_relative 'base'
+
+module Whois
+  class Parsers
+    # Parser for whois.nic.club server.
+    class WhoisNicClub < Base
```

Comment:
The whois.nic.club is probably a `BaseIcannCompliant` parser.

Reply:
Right, looks like it is almost compliant now. It wasn't the case at all when I first wrote this parser ;)

Reply:
Thanks!

I'll merge it in the next few days.